hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -545,7 +545,17 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
   }
 
   @Override
-  public String getBucketLayoutVersion() {
-    return bucket.getMetadata().get(OMConfigKeys.OZONE_OM_LAYOUT_VERSION);
+  public boolean isFSOptimizedBucket() {
+    boolean layoutVersionEnabled =
+            StringUtils.equalsIgnoreCase(
```

Comment:
Lets add a comment here and around the original variable that Layout Version V1 in OM implies optimized FS Paths

Reply:
Thanks a lot @mukul1987 for the review comments. I have updated this in my latest commit.
I'm planning one more follow-up jira task to modify config "`ozone.om.layout.version`" name to something more readable which reflects feature name correctly :-)